src/sickle.ts

```diff
@@ -82,12 +82,10 @@ class Annotator {
 
         if (ctor.body.statements.length) {
           let firstStmt = ctor.body.statements[0];
-          this.emit(
-              ctor.body.getSourceFile().getText().substring(
-                  firstStmt.getFullStart(), ctor.body.getEnd()));
+          this.emit(this.file.text.substring(firstStmt.getFullStart(), ctor.body.getEnd()));
```

Comment:
maybe extract `this.file.text` into a method, and add a comment to it explaining this? It's pretty subtle...